### ISSUE TITLE
documentation on metadata linkage between assay data and clinical and biospecimen data

### ISSUE THREAD

**Issue by ychae** [Sage-Bionetworks/HTAN-data-ingress-documentation-draft#12]
We don't explicitly state anywhere in the docs that centers need to provide the linkage between their assay data and clinical/biospecimen data via metadata. This has been causing confusion and probably should live in the docs. 

Timeline: 1-2 weeks

Related to this issue in data curator: https://github.com/Sage-Bionetworks/data_curator/issues/61

**Comment by xdoan** [comment]
Is this related to how I wanted to make a diagram to illustrate a "complete chain" of metadata and data?

**Comment by milen-sage** [comment]
Yes, definitely related to the information in the diagram. But meanwhile I wanted to add concrete steps on how people should fill in biospecimen/clinical data associated with assay data. 

Someone did this for example:
- uploaded WGS data
- uploaded WGS metadata
- wanted to upload biospecimen data associated w/ the WGS data
- selected the WGS dataset folder and the biospecimen template
- tried to generate the biospecimen template 
- the existing WGS metadata manifest collides with the biospecimen component (along the lines of https://github.com/Sage-Bionetworks/data_curator/issues/61); the app disconnects since the code handling the error is not yet in.

So, I wanted to add a couple of sentences in the documentation outlining the correct steps, so that they
- upload WGS data
- upload WGS metadata
- to upload biospecimen/clinical data associated w/ the WGS data
--- select the biospecimen/clinical dataset folder (if it exists; otherwise create an empty one on Synapse) 
--- select the biospecimen/clinical template
- generate the biospecimen/clinical template
